src/DependencyInjection/HttplugExtension.php

```diff
@@ -382,7 +383,12 @@ private function configureClient(ContainerBuilder $container, $clientName, array
         $serviceId = 'httplug.client.'.$clientName;
 
         if (method_exists($container, 'registerAliasForArgument')) {
-            $container->registerAliasForArgument($serviceId, HttpClient::class, $clientName);
+            $alias = $container->registerAliasForArgument($serviceId, HttpClient::class, $clientName);
+            if (interface_exists(ClientInterface::class)) {
```

Comment:
i think the existence of the psr interface is not the correct way to know which version of httplug we are using. we should additionally check if the httplug client implements the psr interface or something else that tells us if we are installed with a httplug version that implements the interface.

Reply:
Does https://github.com/php-http/HttplugBundle/pull/394/commits/0006ec2294224a8fa4ec34b113ab93ddb0bf42fe solve it?